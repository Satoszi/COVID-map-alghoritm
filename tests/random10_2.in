10 10 5 22
random10_2
|.|..|-++.
..+.+..++-
+.|-..|+.-
.+..+.....
-..|++.-+.
-+..+.-+.+
.+|.|+|+.-
.|++.++++-
++++.++..+
.+++.-+.+.
