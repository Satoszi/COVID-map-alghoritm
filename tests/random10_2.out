8 9
9 8
9 6
5 6
2 6
9 5
6 5
0 5
7 4
3 4
0 4
4 3
1 3
9 2
6 2
3 2
0 2
4 1
6 0
5 0
2 0
0 0
